.evergreen/attach.yml

```diff
@@ -0,0 +1,170 @@
+functions:
+  attach-codegen-diff:
+    command: s3.put
+    type: system
+    params:
+      aws_key: ${AWS_ACCESS_KEY_ID}
+      aws_secret: ${AWS_SECRET_ACCESS_KEY}
+      aws_session_token: ${AWS_SESSION_TOKEN}
+      local_files_include_filter: ["ui/${app_dir}/bin/codegen.diff"]
+      remote_file: ${build_variant}/${task_id}/codegen/
+      bucket: evg-bucket-evergreen-ui
+      content_type: text/plain
+      permissions: public-read
+
+  attach-cypress-results:
+    - command: s3.put
+      type: system
+      params:
+        aws_key: ${AWS_ACCESS_KEY_ID}
+        aws_secret: ${AWS_SECRET_ACCESS_KEY}
+        aws_session_token: ${AWS_SESSION_TOKEN}
+        local_files_include_filter: ["ui/${app_dir}/cypress/screenshots/*"]
+        remote_file: ${build_variant}/${task_id}/
```

Comment:
I think this needs an execution since task files can be fetched by execution and this would end up overwriting previous executions